components/automate-deployment/pkg/usermgmt/client/client_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	"google.golang.org/grpc/status"
 
 	local_users_api "github.com/chef/automate/api/interservice/local_user"
-	teams_api "github.com/chef/automate/api/interservice/teams/v2"
+	teams_api "github.com/chef/automate/api/interservice/teams"
```

Comment:
For consistency with your other changes should be
```suggestion
	"github.com/chef/automate/api/interservice/teams"
```